cmd/utils/flags.go

```diff
@@ -658,6 +658,10 @@ var (
 		Name:  "netrestrict",
 		Usage: "Restricts network communication to the given IP networks (CIDR masks)",
 	}
+	DiscoveryURLFlag = cli.StringSliceFlag{
+		Name:  "discovery-urls",
```

Comment:
Might be better to use `discovery.urls` -- in case we want any more flags surrounding discovery later on